File: Solve/A1/A1-019/A1-019.py
a = input()
b = input()
c = input()
cnt = 0

if a == b: cnt += 1
if a == c: cnt += 1
if b == c: cnt += 1

if (cnt==0) :
    print("all different")
elif (cnt == 1):
    print("neither")
else:
    print("all the same")

File: Solve/A1/A1-019/A1-019.c
#include <stdio.h>
int main(){
    int a,b,c;
    const char *x[4] = {"all different","neither","0","all the same"};
    int cnt=0;
    scanf("%d %d %d",&a,&b,&c);
    if(a==b) cnt++;
    if(a==c) cnt++;
    if(b==c) cnt++;
    printf("%s",x[cnt]);
}
